pages/common/asar.md

```diff
@@ -0,0 +1,19 @@
+# asar
+
+> A file archiver for the Electron platform.
```

Comment:
I know this got merged already, but what's the `Electron platform` in this instance, and how does that relate to file archiving? The only _Electron_ I'm aware of is the shell that the _Atom_ editor uses.

I'm confused.